src/test/login-test.ts

```diff
@@ -0,0 +1,151 @@
+import axios, { AxiosResponse } from 'axios';
+import bcrypt from 'bcrypt';
+import { expect } from 'chai';
+import { serverUrl } from '../setup-server';
+import { AppDataSource } from '../data-source.js';
+import { User } from '../entity/User.js';
+
+interface CreateUserInputData {
+  name: string;
+  email: string;
+  birthDate: string;
+  password: string;
+}
+
+interface LoginInputData {
+  email: string;
+  password: string;
+}
+
+async function postLogin(loginInputData: LoginInputData): Promise<AxiosResponse> {
+  const response = await axios.post(`${serverUrl}graphql`, {
```

Comment:
```suggestion
function postLogin(loginInputData: LoginInputData): Promise<AxiosResponse> {
  return axios.post(`${serverUrl}graphql`, {
```

Lembrando que vc pode fazer isso pq o axios.post já retorna uma Promise.

Reply:
Beleza!